### PR TITLE
Add consent form

### PR DESCRIPTION
**Pull request by samanehsan**
After a user logs in, redirect to the consent form. User must agree to terms before proceeding to take the survey:  
screen shot 2016-07-13 at 11 13 16 am

### CODE REVIEW

File: app/controllers/index.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ export default Ember.Controller.extend({
     actions: {
 	submit ( ) {
 	    console.log(this.getProperties('studyId', 'participantId'));
+      this.transitionToRoute('consent');
```
Comment (samchrisinger) [inline app/controllers/index.js]: Please make sure indentation is consistent here.

File: app/components/select-input/template.hbs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <select class="form-control auto-width" onchange={{action (mut value) value="target.value"}}>
-  <option value=null>Please select</option>
+  <option value=null>{{t 'global.selectUnselected'}}</option>
   {{#each options as |option|}}
-    <option value={{option}} selected={{eq value option}}>{{option}}</option>
+    <option value={{option}} selected={{eq value option}}>{{t option}}</option>
```
Comment (samchrisinger) [inline app/components/select-input/template.hbs:4]: Have you tested that using `eq` like this works?

File: app/utils/translationStrings.js [hunk 3]
```diff
@@ -0,0 +1,1018 @@
+const translations = {
```
Comment (samchrisinger) [inline app/utils/translationStrings.js:1]: I'm a little confused about the need for this file. Can you explain a but about what it's used for?

Comment (samanehsan) [inline app/utils/translationStrings.js:1]: `ember-i18n` flattens the json, and in some cases I need access to an entire dictionary (e.g. the qsort items). When refactoring these components into exp-addons, I just included the json I needed in the files directly (e.g. https://github.com/samanehsan/exp-addons/blob/d8563049f3326b4f29afe6b9b99a9b5f16dd8d38/exp-player/addon/components/exp-card-sort/component.js#L6) so I won't need this file.

File: app/components/select-input/template.hbs (hunk 2 above)
Comment (samanehsan) [inline app/components/select-input/template.hbs:4]: It works! Although, currently users cannot go back in the survey so it's not necessary.

**Comment by samanehsan** [comment]
Closing this PR since this route will be replaced with an `exp-component` after integrating ISP with experimenter.

**Pull request closed without merging**